LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2183,6 +2185,64 @@ private static void GetFintekConfiguration(ISuperIO superIO, Manufacturer manufa
     {
         switch (manufacturer)
         {
+            case Manufacturer.ASRock:
+                switch (model)
+                {
+                    case Model.IMB_1222:
```

Comment:
Wait, if you're saying in the PR title that this board is using NCT6116D, why is this under a Fintek configuration? Mistake?